### DISCUSSION TITLE
I cant' open a Bravo Custom Template because is in computer of coworker. (Open from relative path or cloud?=

### DISCUSSION THREAD

**Discussion by jdmelendez** [sql-bi/Bravo#742] [Q&A]
I have a file .pbix, and I open the Bravo Editor to manage dates. A coworker, has donde a custom template calendar and configured in Visual Studio, and attach to .pbix. Since the template is on your computer, I can't open it.

**Is there a way for this template to be loaded with a relative path, or from the cloud?**

**Comment by albertospelta** [comment]
Bravo supports the `Organization Template` feature, which allows custom date templates to be distributed to all devices/users in the organization. You can find all the details at https://docs.sqlbi.com/bravo/features/manage-dates/deploy-organization-templates.

If you want to use a custom template without enabling the `Organization Template` feature than you need to ask your colleague to provide you with the `.package.json` (see "_The dist folder contains a single template package file_" at https://docs.sqlbi.com/bravo/features/manage-dates/customize-date-template) 

Once the `.package.json` file is on your local device, you can add it to Bravo as a `User` template, as shown below.

![image](https://github.com/sql-bi/Bravo/assets/1957788/897d74e9-22c6-4ad3-80a2-b0573497e238)

**Reply by jdmelendez** [reply]
Hi, Thanks for your answer. 

I have played the tutorial and I have a problem. In the Editor Registry doesn't appear SQLBI Folder. What can I do?

![image](https://github.com/sql-bi/Bravo/assets/59265982/20d62a01-0158-4ab7-8267-1c7423416978)

Thanks

**Reply by albertospelta** [reply]
Registry keys are not automatically created by Bravo. They can be created either manually on a single device or automatically distributed organization-wide via Group Policy by an Active Directory domain administrator.

- For manual deployment you can refer to the `CustomTemplatesEnabled` `BuiltInTemplatesEnabled` and `CustomTemplatesOrganizationRepositoryPath` keys here https://docs.sqlbi.com/bravo/configuration/registry
- For deployment via Group Policy you can see https://learn.microsoft.com/en-us/troubleshoot/windows-client/group-policy/create-and-manage-central-store and download the Group Policy ADMX template from https://github.com/sql-bi/Bravo/tree/main/src/Infrastructure/Security/Policies/ADMX
